web/packages/teleport/src/Audit/Audit.tsx

```diff
@@ -28,6 +28,8 @@ import {
 import useTeleport from 'teleport/useTeleport';
 import useStickyClusterId from 'teleport/useStickyClusterId';
 
+import { ExternalCloudAuditCta } from 'teleport/components/ExternalCloudAuditCta';
```

Comment:
nit: rename variables/types from ExternalCloudAudit -> ExternalAuditStorage for consistency